torch/_higher_order_ops/aoti_call_delegate.py

```diff
@@ -0,0 +1,72 @@
+# mypy: allow-untyped-defs
```

Comment:
You've got pyre-strict but mypy: allow-untyped-defs. Let's remove mypy: allow-untyped-defs ?

Reply:
done